### DISCUSSION TITLE
Vulnerability CVE-2016-1000027 in spring-web project

### DISCUSSION THREAD

**Discussion by ptsang99** [redisson/redisson#4904] [Q&A] [answered]
Hello community, 
Recently we capture a security issue in spring-web library. I did some research and it seems this issue is not really coming from spring fwk but it is coming from the way the application works (this involves Java deserialization stuff https://security-tracker.debian.org/tracker/CVE-2016-1000027 and https://www.tenable.com/security/research/tra-2016-20). From my research, it seems redisson also uses spring library.
My question is, does redisson exposes any HTTP endpoint by default? If it does, how can I disable it? The redis version I'm using is v3.16.6
Any help is greatly appreciated. Thanks

**Comment by mrniko** [comment, marked as answer]
Redisson only integrates with Spring. It doesn't use HTTP protocol.

**Reply by ptsang99** [reply]
Thanks for your answer!
